### PR TITLE
Added pvc quota to docs

### PR DESCRIPTION
**Pull request by markturansky**
Cherry pick https://github.com/kubernetes/kubernetes.github.io/pull/1207

@derekwaynecarr

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes.github.io/1213)

### CODE REVIEW

File: docs/admin/resourcequota/index.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ Resource Quota is enforced in a particular namespace when there is a
 
 ## Compute Resource Quota
 
-The total sum of [compute resources](/docs/user-guide/compute-resources) requested by pods
-in a namespace can be limited.  The following compute resource types are supported:
+The total sum of [compute resources](/docs/user-guide/compute-resources) and [storage resources](/docs/user-guide/persistent-volumes)
```
Comment (devin-donnelly) [inline docs/admin/resourcequota/index.md]: Can be limited, as in "these things are often in short supply", or you can now impose admin-specified limits on compute resources?

If the latter, I'd rephrase to "You can limit the total sum of compute resources and storage resources that can be requested in a given namespace."

**Comment by devin-donnelly** [comment]
@markturansky , did you get a chance to look at that review comment?

**Comment by markturansky** [comment]
Sorry for the delay. Updated per your request.
